numpy/linalg/umath_linalg.c.src

```diff
@@ -1092,7 +1092,10 @@ static inline void
     fortran_int info = 0;
     int i;
     /* note: done in place */
-    LAPACK(@cblas_type@getrf)(&m, &m, (void *)src, &m, pivots, &info);
+    if (m != 0) {
+        /* Lapack (at least some versions) do not handle n == 0 */
+        LAPACK(@cblas_type@getrf)(&m, &m, (void *)src, &m, pivots, &info);
```

Comment:
Which versions are we supporting? Even the oldest 3.0.0 version in lapack_lite we're using supports this, but the third m should be `max(1,M)` according to the documentation